### DISCUSSION TITLE
not only telegram

### DISCUSSION THREAD

**Discussion by fab-far** [sly9/VoyagerTelegramBot#30] [Ideas]
Hi guys.  
I'm using your Bot and aready thank you for your effort in the voyager forum.  I would like to ask the possibility to add an additional channel to your "publisher" in order to notify the voyager actions/events to the smartphone even if the 3-4-5G network is missing (in case you are not at your home but on a montain or desert with poor signals) and you cannot use Telegram.  Should be possible add an  MQTT  channel (MQTT.org)  to send the logs events from Voyager ? With MQTT  you can be able to receive logs in LAN on your smartphone using an MQTT consumer (there are dozen of app on the store).  Of course, w/o 3-4-5G network  you need a local brocker: I tried eclipse moequitto that works very fine. 
This is just an idea to improve your project, as the interface with voyager is already in place, so shoud be easly to add just an additional and optional channel to stream the Voyager messsages. 
Regards 
fabrizio

**Comment by sly9** [comment]
Adding more ways to inform user is on our roadmap — even tho we are not sure what the next one would be.

I’m not very familiar with this MQTT protocol right now, probably should look into it
